content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,374 @@
+  a map of the event venue.
+- :guilabel:`Limit Registrations`: if this checkbox is ticked, a limit to the amount of
+  registrations is added to the event, and that desired limit amount **must** be entered in the
+  blank field before :guilabel:`Attendees`.
+- :guilabel:`Badge Dimension`: select a desired dimension for event badges. The options are:
+  :guilabel:`A4 foldable`, :guilabel:`A6`, or :guilabel:`4 per sheet`.
+- :guilabel:`Badge Background`: if desired, click the :guilabel:`Upload your file` button to upload
+  a custom background for event badges.
+
+When the fields in the event form have been adequately filled in, move on to the three tabs at the
+bottom of the event form for further customization.
+
+Those tabs are: :ref:`Tickets <create_events/event_form/tickets>`, :ref:`Communication
+<create_events/event_form/communication>`, :ref:`Questions <create_events/event_form/questions>`,
+and :ref:`Notes <create_events/event_form/notes>`.
+
+.. _create_events/event_form/tickets:
+
+Tickets tab
+-----------
+
+Create custom tickets (and ticket tiers) for events in the :guilabel:`Tickets` tab of an event form.
+
+.. image:: create_events/tickets-tab.png
+   :align: center
+   :alt: A typical tickets tab on an event form in the Odoo Events application.
+
+To create a ticket, click :guilabel:`Add a line` in the :guilabel:`Tickets` tab. Then, enter a name
+for the ticket (e.g. `Basic Ticket` or `VIP`) in the :guilabel:`Name` field.
+
+Next, in the :guilabel:`Product` field, either select a pre-configured event registration product,
+which Odoo creates when the *Events* app is installed, or create a new one directly from this field.
+
+.. important::
+   Upon installing Odoo *Events* a new product type, *Event Ticket*, becomes selectable on product
+   forms. In order for an event registration product to be selectable in the :guilabel:`Tickets` tab
+   of an event form, the event registration product type **must** be set to *Event Ticket*.
+
+.. note::
+   To create a new event registration product directly from this field, type in the name of the new
+   event registration product, and select either :guilabel:`Create` or
+   :guilabel:`Create and edit...` from the resulting drop-down.
+
+   Clicking :guilabel:`Create` creates the event registration product, which can be edited later.
+   Clicking :guilabel:`Create and edit...` creates the event registration product, and allows the user
+   to edit its configurations right away on a :guilabel:`Create Product` pop-up window.
+
+.. tip::
+   Existing event registration products can be modified directly from this field, as well, by clicking
+   the :guilabel:`Internal link ➡️ (right arrow)` icon, located beside the event registration product.
+   Doing so reveals that product's page on a separate page. Then, proceed to make any desired
+   modifications.
+
+Add or modify the cost of the ticket in the :guilabel:`Price` field. Then, determine a
```

Comment:
```suggestion
Add or modify the cost of the ticket in the :guilabel:`Price` field. Then, determine a
```
Does this update the product price in the product catalog?

Reply:
No. The price on the product form in the product catalog is the default...that's why I tried to specify that they could 'modify' it...but perhaps that isn't specific enough. I'll add a note admonition to this explanation to make things more clear in this regard. Thanks for the helpful suggestion 👍